changelog.txt

```diff
@@ -14,6 +14,8 @@
 * Fix - Checkout with block-based themes.
 * Add - UPE payment method - EPS.
 * Fix - Redundant payment intents for UPE.
+* Fix - Replace uses of is_ajax() with wp_doing_ajax().
```

Comment:
Should we add something to this changelog (specifically this one) to say that it relates to subscriptions-core?

Reply:
Done ✅